src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -656,6 +643,71 @@ size_t SceneGadget::objectsAt(
 	return paths.size();
 }
 
+std::optional<V3f> SceneGadget::normalAt( const IECore::LineSegment3f &lineInGadgetSpace ) const
+{
+	V3f centerP;
+	ScenePlug::ScenePath centerPath;
+
+	auto viewportGadget = ancestor<ViewportGadget>();
+
+	V2f centerRasterP = viewportGadget->gadgetToRasterSpace( lineInGadgetSpace.p0, this );
+	centerRasterP.x = (float)( (int)centerRasterP.x ) + 0.5f;
+	centerRasterP.y = (float)( (int)centerRasterP.y ) + 0.5f;
```

Comment:
We don't actually want to add the 0.5 offset here, right? It's added in `objectAt()`, so this would make it a full pixel offset?

Reply:
I'm pretty confident that should be removed, so I did so in https://github.com/GafferHQ/gaffer/pull/5630/commits/5b3a86f0c6f30e29c2d422c7897d1861af5b5329. I can always drop the commit if I have a case of Friday afternoon mind mush.

Reply:
My Thursday morning mind mush concurs with your Friday afternoon mind mush. Thanks for spotting!